packages/graph-engine/tests/suites/nodes/color/matchAlpha.test.ts

```diff
@@ -0,0 +1,103 @@
+import { Graph } from '../../../../src/graph/graph.js';
+import { describe, expect, test } from 'vitest';
+import { getAllOutputs } from '../../../../src/utils/node.js';
+import { isNaN } from 'lodash-es';
```

Comment:
There is a native isNaN we can use

Reply:
That's what I did first but it kept getting the red squiggles, until I used autocomplete and it found this one, then it worked.

Reply:
Did you use `Number.isNaN` ?